src/dlgMapLabel.cpp

```diff
@@ -0,0 +1,197 @@
+/***************************************************************************
+ *   Copyright (C) 2022 by Piotr Wilczynski - delwing@gmail.com            *

```

Comment:
Can you add your copyright to all the files that this PR touches please...

Reply:
Done.